core/src/avm2/globals/Math.as

```diff
@@ -28,9 +28,6 @@ package {
         // This is a hacky way to specify `-Infinity` as a default value.
         private static const NegInfinity: Number = -1 / 0;
         public static native function max(x: Number = NegInfinity, y: Number = NegInfinity, ...rest): Number;
```

Comment:
can this use `-Infinity` now?

Reply:
Unfortunately no. `asc.jar` requires default values to be compile-time constants, which AFAIK can be either a literal or a `const` member.